docs/blog/newsletter-may-2022.rst

```diff
@@ -0,0 +1,118 @@
+
+.. post:: May 02, 2022
+   :tags: newsletter
+
+####################################
+Write the Docs Newsletter – May 2022
+####################################
+
+Hi everybody!
+
+It's Beth here, joining you for my last email as editor. It's been a real privilege to take up space in your inbox these past few years - thank you for reading! I'll let our new editor introduce himself in the next edition, but I'm excited to see someone new taking the reins, and I'm sure he's going to do a fantastic job.
+
+In community news, hopefully all of you going to the virtual Portland conference have seen that the `schedule </conf/portland/2022/news/announcing-schedule/>`__ is out, but sharing the link again just in case. For those of you more in my European neck of the woods, you might like to know that we've announced `dates for the virtual Prague conference </conf/prague/2022/news/welcome/>`__: 11th-13th September. And if you think you have some ideas that you might like to share - please check out the `Call for Proposals </conf/prague/2022/news/announcing-cfp/>`__, and consider giving a talk. The CFP is open til 30th June.
+
+And with that, let's see what we've learnt on Slack during April!
+
+-------------------
+Link-checking tools
+-------------------
+
+This month, a question about useful tools for checking links in HTML output prompted several suggestions. Some have built custom link-checking tools, but there are plenty of ready-to-use open-source options too. Here's the roundup:
+
+* `htmltest <https://github.com/wjdp/htmltest>`_ was most suggested, based on fast performance and configurability. htmltest is also especially suitable for link testing in continuous integration environments -- you can check links in HTML files in a folder, with no need to run a web server.
+- `Muffet <https://github.com/raviqqe/muffet>`_ is another fast and thorough link-checking tool, with pretty-printed or JSON output. There's also a GitHub Action that uses Muffet, `My Broken Link Checker <https://github.com/ruzickap/action-my-broken-link-checker>`_, although you may have to adjust Muffet's maximum requests per second to accommodate GitHub rate limits.
+- `HTMLProofer <https://github.com/gjtorikian/html-proofer>`_ validates all HTML output, including internal and external links, image references and alt tags, and scripts. You can run HTMLProofer on one file, a directory, and arrays of directories and links, and it's also highly configurable.
+- `LinkChecker <https://github.com/linkchecker/linkchecker/>`_ gets the job done, and it's documented! Find detailed information and configuration options at the `LinkChecker documentation site <https://linkchecker.github.io/linkchecker/index.html>`_.
```

Comment:
```suggestion
* `Muffet <https://github.com/raviqqe/muffet>`_ is another fast and thorough link-checking tool, with pretty-printed or JSON output. There's also a GitHub Action that uses Muffet, `My Broken Link Checker <https://github.com/ruzickap/action-my-broken-link-checker>`_, although you may have to adjust Muffet's maximum requests per second to accommodate GitHub rate limits.
* `HTMLProofer <https://github.com/gjtorikian/html-proofer>`_ validates all HTML output, including internal and external links, image references and alt tags, and scripts. You can run HTMLProofer on one file, a directory, and arrays of directories and links, and it's also highly configurable.
* `LinkChecker <https://github.com/linkchecker/linkchecker/>`_ gets the job done, and it's documented! Find detailed information and configuration options at the `LinkChecker documentation site <https://linkchecker.github.io/linkchecker/index.html>`_.
```

At least, I think that's what the warning is about, but I'm no expert on the syntax :shrug: